assignments/assignment01.rb

```diff
@@ -69,10 +102,50 @@ def my_reverse(s)
 # that determines whether a string is a palindrome
 def palindrome?(s)
   # your implementation here
+  def toChars(s)
```

Comment:
You could implement this much more simply using built in String methods. You could use delete or gsub to get rid of non-alpha characters.